css/modern.css

```diff
@@ -263,6 +336,21 @@ fieldset div div div label span {
     background: var(--background-color);
 }
 
+._pendo-step-container-styles {
+    background-color: var(--accent) !important;
+}
+.pendo-mock-flexbox-element:nth-child(2)>button {
```

Comment:
Does this selector have to be this specific? It's okay if it does, but would it break things if it were changed to say `pendo-mock-flexbox-element > button` or even `pendo-mock-flexbox-element button`?

Reply:
iirc that was for the pendo box in 266e0b2, that was for the second one only since only the second one needed to be changed.

Reply:
Got it. Makes sense.